OpenEdXMobile/res/layout/section_row_layout.xml

```diff
@@ -59,6 +61,7 @@
         android:id="@+id/rv_sub_section"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/edx_divider_length"
```

Comment:
In expended view last-child bottom line get overlapped with tapped effect

Reply:
It's working fine for me plus I did fix it before. 🤔